### PR TITLE
Fix/general fixes

### PR DESCRIPTION
**Pull request by Edunknown**
# PR Summary: Code Abstraction and Component Documentation Enhancements 📝

## Overview:

This PR focuses on improving code abstraction and adding detailed documentation across the project to enhance code readability and maintainability. Additionally, several fixes have been implemented to address specific issues related to transaction fetching and responsive design.

### 1. Component Documentation 🗒️: 

- Added detailed comments to all components to provide better context and understanding for future developers.
- Documented utility functions and services to ensure clarity in their purpose and usage, improving overall developer experience.

### 2. Refactorization: 

Refactorized most components that could be refactorized.

## Fixes 🩹:

- Resolved an issue in the "Bridge Stats" component on the homepage where transactions were being fetched twice, leading to duplicate entries.
- Made visual adjustments to improve the application's responsiveness and user interface across different screen sizes.

### CODE REVIEW

File: src/pages/components/Home/BridgeStatsComponent.tsx [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
- * @description This component is a reusable component that displays the bridge stats of the home page.
+ * @description Component that displays the bridge stats.
  *
  * @returns { JSX.Element }
  */
 
 const BridgeStatsComponent: FC = () => {
 	const [tbtcSupply, setTbtcSupply] = useState<number>(0);
 	const [tbtcTransactions, setTbtcTransactions] = useState<Array<any>>([]);
+	console.log('🚀 ~ tbtcTransactions:', tbtcTransactions);
```
Comment (JSanchezFDZ) [inline src/pages/components/Home/BridgeStatsComponent.tsx]: Quitar los console log

File: src/services/tbtcServices.ts [hunk 2]
```diff
@@ -415,7 +419,11 @@ export const getTbtcTransactions = async (
 	const {
 		data: { result },
 	} = await axios.get(url);
+	console.log('🚀 ~ result:', result);
```
Comment (JSanchezFDZ) [inline src/services/tbtcServices.ts]: Borrar

**Review by JSanchezFDZ — CHANGES_REQUESTED** [review]

**Review by JSanchezFDZ — APPROVED** [review]
LGTM!

**Comment by JSanchezFDZ** [comment]
@evandrosaturnino here is the refactored code + some fixes + commented code.

I'll let you check it so it doesn't break anything in your current version.
With this I think it should be complete.

**Review by evandrosaturnino — APPROVED** [review]
LGTM

**Pull request merged**